Fallout2/Fallout1in2/mods/fo1_lang_german/text/german/dialog/MASTER.MSG

```diff
@@ -1,129 +1,121 @@
 {100}{}{Du siehst den Meister der Supermutanten.}
 {101}{Mast00}{Du hast die Wahl: Trittst du der Einheit bei, oder stirbst du hier? Beitritt! Tod! Beitritt! Tod!}
 {102}{}{Ich kann eh nur verlieren, also tret' ich der Einheit bei.}
-{103}{}{Wenn du mir beweist, dass eure Einheit das Beste f�r die Menschheit ist, dann helfe ich euch.}
-{104}{}{Wie w�r's, wenn ich dich einfach umlege und abhaue?}
-{105}{}{Heute wird jemand sterben, aber ich bin's nicht! Ich habe n�mlich einen Plan.}
+{103}{}{Wenn du mir beweist, dass eure Einheit das Beste für die Menschheit ist, dann helfe ich euch.}
+{104}{}{Wie wär's, wenn ich dich einfach umlege und abhaue?}
```

Comment:
Did you test if those changes are working ingame?

Reply:
I don't even have Fallout Et Tu on my machine, but in theory they should work.

Reply:
Theory doesn't belong here ... It needs to be tested. Fallout is very picky about the file encoding.